### PR TITLE
Mark Deprecated endpoints in the docs

### PR DESCRIPTION
**Pull request by fundthmcalculus**
* I looked for discussion of `Issue` endpoint, and everything seems to talk about issuing from templates.
* Added a warning header about deprecation

### CODE REVIEW

File: docs/reference/services/credential-service.md [hunk 1]
```diff
@@ -12,6 +12,9 @@ The Credential Service exposes functionality for issuance, proof generation, ver
 
 ## Issue Credential
 
+!!! warning "Deprecation Notice"
+    This endpoint is deprecated, and will be removed on May 1, 2023. Please call [Issue Credential From Template](./credential-service.md#issue-credential-from-template).
```
Comment (janpieterz) [inline docs/reference/services/credential-service.md:16]: To do this, we will need to release a new SDK for all languages. Which is long overdue anyway.

There might be some other methods or parameters that are deprecated as well. I know we should have done this before, but would you be able to add a similar note for those?
The ones that jumps to mind is the Invite one (not sure if others).

**Review by janpieterz — APPROVED** [review]

File: docs/reference/services/credential-service.md (hunk 1 above)
Comment (janpieterz) [reply docs/reference/services/credential-service.md:16]: The dates for all of those should be the same FYI. So we might need to rewrite some comments in `server-api`

Comment (fundthmcalculus) [reply docs/reference/services/credential-service.md:16]: Agreed. I'll leave this ticket open while we decide, and make it a single update for the month.

Comment (fundthmcalculus) [reply docs/reference/services/credential-service.md:16]: `ProviderService.Invite` isn't currently in the docs at all, so we're good on that front.

Comment (fundthmcalculus) [reply docs/reference/services/credential-service.md:16]: Marked `ProviderService.Update` as deprecated

**Pull request merged**